src_c/key.c

```diff
@@ -439,6 +439,11 @@ static const char *SDL1_scancode_names[SDL_NUM_SCANCODES] = {
     "AudioFastForward",
 };
 
+#if defined(BUILD_STATIC)
+// from SDL_keyboard.c
+extern char *
+SDL_UCS4ToUTF8(Uint32 ch, char *dst);
+#else
 /* Taken from SDL_iconv() */
 char *
 SDL_UCS4ToUTF8(Uint32 ch, char *dst)
```

Comment:
ps. We should never define our own "SDL_" functions. This has bitten us in the past. Instead we should create pg_XXX functions which inside may call the SDL_XXX function, or include the function body. This way if someone links something with the SDL_XXX function in the future we have no problems.